operator/cmd/mesh/root.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	SetFlagHelpStr = `Set a value in IstioOperator CustomResource. e.g. --set policy.enabled=true.
-Overrides the corresponding path value in the selected profile or passed through IstioOperator CR
-customization file`
+	SetFlagHelpStr = `Override an IstioOperator value, e.g. to choose a profile
+(--set profile=demo), enable or disable components (--set policy.enabled=true), or override Istio 
```

Comment:
I think this is `-- components.policy.enabled=true` now?

Reply:
Yes you're right, good catch. Will change